README.md

```diff
@@ -333,6 +281,16 @@ my-cluster-kafka-bootstrap    ClusterIP   172.30.228.90    <none>        9091/TC
 
 In this case the Kafka Cluster service is `my-cluster-kafka-bootstrap:9092`.
 
+Or run the following command to take the custom resource name:
```

Comment:
Also here :

"Or run the following command to retrieve the Kafka custom resource name:"
or
"Or run the following command to retrieve the Kafka instance name:"